scrapers/Mandyflores.yml

```diff
@@ -0,0 +1,29 @@
+name: Mandyflores
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - mandyflores.com/vod/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $updateDesc: //span[@class="update_description"]
+      $divCenter: /div[@align="center"]
+    scene:
+      Title: //span[@class="title_bar_hilite"]
+      Details:
+        selector: $updateDesc$divCenter/span/span[@style]/text()[normalize-space(.)]|($updateDesc | $updateDesc/p)/text()[normalize-space(.)]|($updateDesc$divCenter/text())[1]|($updateDesc/text())[1]
+        concat: "\n"
+      Date:
+        selector: //div[@class="cell update_date"][not(ancestor::span[@class="update_description"])]/text()

```

Comment:
in some cases the selector returns more than one dates even with the ancestor limitation ( `https://mandyflores.com/vod/scenes/Mistletoe-Kiss-Attack_vids.html` ). The first is always ok though so can keep only that ?
`//div[@class="cell update_date"][not(ancestor::span[@class="update_description"])]/text()[1]`

Reply:
Oh man, I love the structure on this site. Good catch